packages/integration-tests/src/fixtures/hosts/index.ts

```diff
@@ -7,10 +7,13 @@ const hostIds = [
   '1e40aa5d-0b69-4122-a562-bff1e35e7071',
   '2e40aa5d-0b69-4122-a562-bff1e35e7072',
   '3e40aa5d-0b69-4122-a562-bff1e35e7073',
+  '4e40aa5d-0b69-4122-a562-bff1e35e7074',
+  '5e40aa5d-0b69-4122-a562-bff1e35e7075',
 ];
-const hostIPs = ['192.168.122.0/24', 'IP2', 'IP3'];
 
-const discoveredHosts = [hostDiscover(0), hostDiscover(1), hostDiscover(2)];
+const hostIPs = ['192.168.122.0/24', 'IP2', 'IP3', 'IP4', 'IP5'];
+
+const discoveredHosts = [hostDiscover(0), hostDiscover(1), hostDiscover(2), hostDiscover(3), hostDiscover(4)];
```

Comment:
```suggestion
const discoveredHosts = Array(5).fill(hostDiscover).map((f, idx) => f(idx));
```
:)~

Reply:
Did a slight variation, but same idea to not DRY.

Reply:
For some reason it wasn't updating this branch, I added it to Read Only cluster story
https://github.com/openshift-assisted/openshift-assisted-ui/pull/18/commits/336754bb39fa31b8d64daba7c48ae895d704b60a